pkg/semtech-loramac/include/semtech-loramac/timer.h

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2017 Fundacion Inria Chile
+ *               2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         pkg_semtech-loramac
+ * @brief           Semtech LoRaMAC timer compatibility definitions
+ * @{
+ *
+ * @file
+ *
+ * @author          José Ignacio Alamos <jialamos@uc.cl>
+ * @author          Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef SEMTECH_LORAMAC_TIMER_H
+#define SEMTECH_LORAMAC_TIMER_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "xtimer.h"
+#include "msg.h"
+
+#include "semtech_loramac.h"
+
+/**
+ * @brief   Timer object description
+ */
+typedef struct TimerEvent_s
+{
+    uint32_t timeout;   /**< Timer timeout in us */
+    uint8_t running;    /**< Check if timer is running */
+    xtimer_t dev;       /**< xtimer instance attached to this LoRaMAC timer */
+    msg_t msg;          /**< message attacher to this LoRaMAC timer */
+    void (*cb)(void);   /**< callback to call when timer timeout */
+} TimerEvent_t;
```

Comment:
Coding convention (`slug_case` instead of CamelCase; also the opening `{` is of)?

Reply:
This is normal since this file is included by the internal LoRaMAC patched code.

Reply:
I can fix the opening `{` though

Reply:
Ah, alright then!